proxy/http/HttpSM.cc

```diff
@@ -4668,6 +4668,107 @@ HttpSM::send_origin_throttled_response()
   call_transact_and_set_next_state(HttpTransact::HandleResponse);
 }
 
+static void
+set_tls_options(NetVCOptions &opt, OverridableHttpConfigParams *txn_conf)
+{
+  // remove before 9.0.0 release
+  // Backwards compatibility if proxy.config.ssl.client.verify.server is explicitly set
+  RecSourceT source             = REC_SOURCE_DEFAULT;
+  bool set_backwards_compatible = false;
+  if (RecGetRecordSource("proxy.config.ssl.client.verify.server", &source, false) == REC_ERR_OKAY) {
+    if (txn_conf->ssl_client_verify_server || (source != REC_SOURCE_DEFAULT && source != REC_SOURCE_NULL)) {
+      // Assuming that conf override is using the old name only if the base
+      // config is.
+      int8_t verifyServer = 0;
+      if (txn_conf->ssl_client_verify_server) {
+        verifyServer = txn_conf->ssl_client_verify_server;
+      } else {
+        REC_EstablishStaticConfigByte(verifyServer, "proxy.config.ssl.client.verify.server");
```

Comment:
This is done potentially on every upstream connection? That seems somewhat risky.

Reply:
For performance concerns?  Or reload concerns?  We could move it into the HttpConfig object I suppose.

Reply:
Alan clarified that I was using a call that read the value and set a callback.  Changed the call to just read the value.  No callback setting.